pkg/controller/activemqartemis/mocks_test.go

```diff
@@ -0,0 +1,148 @@
+package activemqartemis
+
+import (
+	brokerv2alpha1 "github.com/rh-messaging/activemq-artemis-operator/pkg/apis/broker/v2alpha1"
+	"github.com/rh-messaging/activemq-artemis-operator/pkg/resources/pods"
+	"github.com/rh-messaging/activemq-artemis-operator/pkg/utils/namer"
+	"github.com/rh-messaging/activemq-artemis-operator/pkg/utils/selectors"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+)
+
+var NameBuilder namer.NamerData
+var labels = selectors.LabelBuilder.Labels()
+var (
```

Comment:
Either use a var block, or individual var declarations. No good reason to do it one way for the first two and another way for the other 3-4.

Reply:
added inside var block.